src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -0,0 +1,208 @@
+(ns quo.components.profile.collectible-list-item.view
+  (:require
+    [quo.components.avatars.collection-avatar.view :as collection-avatar]
+    [quo.components.counter.collectible-counter.view :as collectible-counter]
+    [quo.components.icon :as icon]
+    [quo.components.list-items.preview-list.view :as preview-list]
+    [quo.components.markdown.text :as quo]
+    [quo.components.profile.collectible-list-item.style :as style]
+    [quo.foundations.colors :as colors]
+    [quo.foundations.gradients :as gradients]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]
+    [utils.i18n :as i18n]))
+
+(defn- fallback
```

Comment:
The word `fallback` kinda vague here. Maybe `fallback-view`

Reply:
sure, can adjust